inc/functions/files.php

```diff
@@ -1051,7 +1051,9 @@ function rocket_find_wpconfig_path() {
  * @return string The footprint that will be printed
  */
 function get_rocket_footprint( $debug = true ) {
-	$footprint = "\n" . '<!-- This website is like a Rocket, isn\'t it? Performance optimized by ' . WP_ROCKET_PLUGIN_NAME . '. Learn more: https://wp-rocket.me';
+	$footprint = defined( 'WP_RWL' ) ? 
```

Comment:
I think we should use a more specific constant name here, and also be consistent with our other constants.

`WP_ROCKET_WHITE_LABEL_FOOTPRINT' is more descriptive of what's for.

Reply:
Done :+1: